src/slc/xliff/xliff.py

```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
+from pdb import Pdb
```

Comment:
@erral  This is probably not intended… (No harm done, since it's not used anywhere)